uxarray/grid/grid.py

```diff
@@ -1506,6 +1506,18 @@ def global_sphere_coverage(self):
         (i.e. contains no holes)"""
         return not self.partial_sphere_coverage
 
+    @property
+    def inverse_face_indices(self):
+        if self._is_subset:
+            return self._ds["inverse_face_indices"]
```

Comment:
Please add a docstring. I would suggest throwing an exception if the grid is not a subset.